src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -22,6 +22,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">true</EnableDefaultNoneItems>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="@(Compile)" Condition=" '$(EnableDefaultCompileItems)' != 'true' And '$(EnableDefaultNoneItems)' == 'true' " />
```

Comment:
Can this go with the rest of the removes?

https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props#L26